build/media-suite_deps.sh

```diff
@@ -0,0 +1,107 @@
+SOURCE_REPO_UAVS3D=https://github.com/uavs3/uavs3d.git
+SOURCE_REPO_LIBZEN=https://github.com/MediaArea/ZenLib.git
+SOURCE_REPO_LIBMEDIAINFO=https://github.com/MediaArea/MediaInfoLib.git
+SOURCE_REPO_MEDIAINFO=https://github.com/MediaArea/MediaInfo.git
+SOURCE_REPO_VIDSTAB=https://github.com/georgmartius/vid.stab.git
+SOURCE_REPO_DLFCN=https://github.com/dlfcn-win32/dlfcn-win32.git
+SOURCE_REPO_FREI0R=https://github.com/dyne/frei0r.git
+SOURCE_REPO_DECKLINK=https://gitlab.com/m-ab-s/decklink-headers.git
+SOURCE_REPO_LIBMFX=https://github.com/lu-zero/mfx_dispatch.git
+SOURCE_REPO_AMF=https://github.com/GPUOpen-LibrariesAndSDKs/AMF.git
+SOURCE_REPO_GPAC=https://github.com/gpac/gpac.git
+SOURCE_REPO_SVTHEVC=https://github.com/OpenVisualCloud/SVT-HEVC.git
+SOURCE_REPO_SVTAV1=https://gitlab.com/AOMediaCodec/SVT-AV1.git
+SOURCE_REPO_SVTVP9=https://github.com/OpenVisualCloud/SVT-VP9.git
+SOURCE_REPO_XVC=https://github.com/divideon/xvc.git
+SOURCE_REPO_X264=https://code.videolan.org/videolan/x264.git
+SOURCE_REPO_FFMS2=https://github.com/FFMS/ffms2.git
+SOURCE_REPO_LIBLSMASH=https://github.com/l-smash/l-smash.git
+SOURCE_REPO_X265=https://bitbucket.org/multicoreware/x265_git.git
+SOURCE_REPO_XVID=https://github.com/m-ab-s/xvid.git
+SOURCE_REPO_FFNVCODEC=https://git.videolan.org/git/ffmpeg/nv-codec-headers.git
+SOURCE_REPO_SRT=https://github.com/Haivision/srt.git
+SOURCE_REPO_LIBRIST=https://code.videolan.org/rist/librist.git
+SOURCE_REPO_LENSFUN=https://github.com/lensfun/lensfun.git
+SOURCE_REPO_VVC=https://gitlab.com/m-ab-s/VVCSoftware_VTM.git
+SOURCE_REPO_UVG266=https://github.com/ultravideo/uvg266.git
+SOURCE_REPO_LIBVVENC=https://github.com/fraunhoferhhi/vvenc.git
+SOURCE_REPO_LIBVVDEC=https://github.com/fraunhoferhhi/vvdec.git
+SOURCE_REPO_AVISYNTH=https://github.com/AviSynth/AviSynthPlus.git
+SOURCE_REPO_VULKANLOADER=https://github.com/KhronosGroup/Vulkan-Loader.git
+SOURCE_REPO_LIBPLACEBO=https://code.videolan.org/videolan/libplacebo.git
+SOURCE_REPO_LUAJIT=https://github.com/LuaJIT/LuaJIT.git
+SOURCE_REPO_ANGLE=https://chromium.googlesource.com/angle/angle
+SOURCE_REPO_MUJS=https://github.com/ccxvii/mujs.git
+SOURCE_REPO_MRUBY=https://github.com/mruby/mruby.git
+SOURCE_REPO_SHADERC=https://github.com/google/shaderc.git
+SOURCE_REPO_GLSLANG=https://github.com/KhronosGroup/glslang.git
+SOURCE_REPO_SPIRV_CROSS=https://github.com/KhronosGroup/SPIRV-Cross.git
+SOURCE_REPO_SPIRV_TOOLS=https://github.com/KhronosGroup/SPIRV-Tools.git
+SOURCE_REPO_SPIRV_HEADERS=https://github.com/KhronosGroup/SPIRV-Headers.git
+SOURCE_REPO_MPV=https://github.com/mpv-player/mpv.git
+SOURCE_REPO_LIBMXF=https://gitlab.com/m-ab-s/libmxf.git
+SOURCE_REPO_LIBMXFPP=https://gitlab.com/m-ab-s/libmxfpp.git
+SOURCE_REPO_LIBBMX=https://gitlab.com/m-ab-s/bmx.git
+SOURCE_REPO_NEON=https://github.com/notroj/neon.git
+SOURCE_REPO_LIBMUSICBRAINZ=https://github.com/wiiaboo/libmusicbrainz.git
+SOURCE_REPO_CYANRIP=https://github.com/cyanreg/cyanrip.git
```

Comment:
perhaps it may be better to sort these alphabetically to be able to quickly look for a repo, so `sort -uo media-suite_deps.sh{,}`

Reply:
I was unsure. The list was "in order of appearance".

But it's sorted now.